LoopKit/InsulinKit/PersistedPumpEvent.swift

```diff
@@ -28,6 +28,8 @@ public struct PersistedPumpEvent {
     public let type: PumpEventType?
     /// Whether the pump event is marked mutable
     public let isMutable: Bool
+    /// Whether the event was automatic or manually triggered
```

Comment:
[nit] Consider adding to the code comment what `nil` would mean for this (i.e. what's the default?)